es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchRestDAOV5.java

```diff
@@ -218,8 +218,10 @@ public void setup() throws Exception {
 
         try {
             initIndex();
-            updateIndexName();
-            Executors.newScheduledThreadPool(1).scheduleAtFixedRate(this::updateIndexName, 0, 1, TimeUnit.HOURS);
+            if (config.isTaskExecLogIndexingEnabled()) {
```

Comment:
Generic question - Which implementation we use `ElasticSearchDAOV5` or `ElasticSearchRestDAOV5`? And whats the other implementation for?